### DISCUSSION TITLE
Fazer um grupo no Telegram?

### DISCUSSION THREAD

**Discussion by Marcelosilva10** [frontendbr/forum#146] [Discussão]
Pessoal  que vocês acham de fazer um grupo no Telegram, aonde pode ser discutir os eventos, atuais ou vagas ou falar sobre novas tecnologias ou arquivos em PDF?

O que vocês acham?

**Comment by icaromh** [comment]
Eu particularmente não gosto. Só Slack e olhe lá.
Eu deixo todos os grupos no mudo no celular, é um pé no saco e fica chamando atenção o dia inteiro - cada pessoa tem um horário diferente disponível para enviar mensagens.

Vou dar uma de anti-social. Não oponho fazerem, mas não entraria :disappointed:

**Comment by fabricionaweb** [comment]
Acho que já tem coisa de mais... Facebook, GitHub, Slack...

**Comment by Marcelosilva10** [comment]
A sim sem problemas foi só uma opinião mesmo tem razão...

**Comment by felipefialho** [comment]
Então @Marcelosilva10, muita coisa foi debatida no começo dessa org.

Então em tese, temos algumas frentes (sem ligação entre os administradores das mesmas).
- [Facebook](https://www.facebook.com/groups/frontendbrasil/?ref=bookmarks) (aka @suissa) - Divulgação e compartilhamento de artigos, projetos, etc
- [Slack](http://frontendbrasil.slack.com/) - Dúvidas rápidas, bate papo mais informal
- [Github](https://github.com/frontendbr/forum) - Debates e questões que requerem mais aprofundamento, para gerar escalabilidade na internet.

Acho que está de bom tamanho :)

**Comment by Marcelosilva10** [comment]
@LFeh estou no grupo do @suissa do Be-mean, mais muito obrigado por avisar.

**Comment by skyzito** [comment]
Temos um grupo no telegram da galera de Python MG e acho bem interessante , principalmente por conta dos encontros e jogar conversa fora. Se Rolar eu to nessa ai :)
